### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by kcords**
Completed modal layout and basic functionality. Working transition animations and icon buttons will be implemented as separate feature branches.

### CODE REVIEW

File: client/src/components/Button/Button.jsx [hunk 1]
```diff
@@ -4,10 +4,16 @@ import PropTypes from 'prop-types';
 import classes from './Button.module.css';
 
 function Button (props) {
-  const {children, type} = props;
+  const { children, type, modalOpen, setModalOpen } = props;
+
+  const handleClick = () => {
+    console.log('clicked', setModalOpen)
```
Comment (juanoxmar) [inline client/src/components/Button/Button.jsx]: Remove console.log

File: client/src/components/PhotoGrid/PhotoGrid.jsx [hunk 2]
```diff
@@ -1,64 +1,110 @@
...
+
+      <Modal modalOpen={modalOpen}>
+        <PhotoViewer
+          modalOpen={modalOpen}
+          setModalOpen={setModalOpen}
+          selectedPhoto={selectedPhoto}
+          setSelectedPhoto={setSelectedPhoto}
+          photos={photos}
+        />
+        {/* <Button modalOpen={modalOpen} setModalOpen={setModalOpen}>Close</Button>
```
Comment (juanoxmar) [inline client/src/components/PhotoGrid/PhotoGrid.jsx]: Remove Comments

File: client/src/components/PhotoGrid/PhotoGrid.jsx [hunk 3]
```diff
@@ -1,64 +1,110 @@
...
+    getPhotos(listingId);
+  }, []);
+
+  // const test = () => console.log('Made it!'); //! THIS WILL BE REMOVED
+
   return (
     <div className={classes.photogrid}>
-      {/* {photos.slice(0, 5).map((photo) => (
+      {/* {photos.slice(0, 5).map((photo) => ( //!THIS SECTION WILL BE REMOVED
         <PhotoItem
```
Comment (juanoxmar) [inline client/src/components/PhotoGrid/PhotoGrid.jsx]: Remove Comments

**Review by juanoxmar — CHANGES_REQUESTED** [review]
See Comments

File: client/src/components/Button/Button.jsx (hunk 1 above)
Comment (kcords) [reply client/src/components/Button/Button.jsx]: This has been removed.

File: client/src/components/PhotoGrid/PhotoGrid.jsx (hunk 3 above)
Comment (kcords) [reply client/src/components/PhotoGrid/PhotoGrid.jsx]: This code block has been removed.

File: client/src/components/PhotoGrid/PhotoGrid.jsx (hunk 2 above)
Comment (kcords) [reply client/src/components/PhotoGrid/PhotoGrid.jsx]: This has been removed.

**Review by juanoxmar — APPROVED** [review]

**Pull request merged**